bundles/mapping/mapmodule/MapModuleClass.ol.js

```diff
@@ -378,7 +380,56 @@ export default class MapModule extends AbstractMapModule {
 
         return olExtent.containsCoordinate(extent, lonlatArray);
     }
+    getLocationGeoJSON (position) {
+        const coord = [position.lon, position.lat];
+        const accuracy = position.accuracy;
+        var features = [];
+        features.push(new olFeature({
+            geometry: new olGeom.Point(coord),
+            name: 'location'
+        }));
+        if (accuracy) {
+            features.push(new olFeature({
+                geometry: fromCircle(new olGeom.Circle(coord, accuracy), 50),
+                name: 'accuracy'
+            }));
+        }
+        return this.getGeoJSONFromFeatures(features);
+    }
+    updateLocationPath (coordinate) {
+        if (!Array.isArray(coordinate)) {
+            return;
+        }
 
+        if (!this._locationPath) {
+            this._locationPath = new olGeom.LineString([coordinate]);
+            return null;
+            // or return point
+            // return this.getLocationGeoJSON(coordinate[0], coordinate[1]);
+        }
+        this._locationPath.appendCoordinate(coordinate);
+        return this.getLocationPathGeoJSON();
+    }
+    getLocationPathGeoJSON () {
+        var geom = this._locationPath;
+        if (geom) {
+            var feature = new olFeature({
+                geometry: geom,
+                name: 'path'
+            });
+            return this.getGeoJSONFromFeatures([feature]);
+        }
+        return null;
+    }
+    getLocationPathCoordinates () {
+        if (this._locationPath) {
+            return this._locationPath.getCoordinates();
+        }
+        return [];
+    }
+    clearLocationPath () {
+        this._locationPath = null;
+    }
     /* <------------- / OL3 specific ----------------------------------- */
```

Comment:
Do we need to keep track of the `_locationPath` in the MapModule impl? It's already too big and sprawling.